### ISSUE TITLE
Add support for VS2022

### ISSUE THREAD

**Issue by julienalofs** [conan-io/cmake-conan#367]
https://github.com/conan-io/cmake-conan/blob/43e385830ee35377dbd2dcbe8d5a9e750301ea00/conan.cmake#L60
should be
```
elseif(NOT MSVC_VERSION VERSION_LESS 1930 AND MSVC_VERSION VERSION_LESS 1940)
        set(${result} 17 PARENT_SCOPE)
else()
```

**Comment by czoido** [comment]
Hi @julienalofs,
Thanks for pointing this out, we'll update this for the next release.

**Comment by czoido** [comment]
Hi @julienalofs,
I have just checked and Visual Studio 2022 support was already added in https://github.com/conan-io/cmake-conan/pull/350
I'm closing this issue, please feel free to reopen if you have any question.

**Comment by kernhanda** [comment]
When will we get a tagged release for this? Otherwise, we're left to use the development branch's conan.cmake

**Comment by czoido** [comment]
Hi @kernhanda,
We have just released version 0.17. with those changes included: https://github.com/conan-io/cmake-conan/releases/tag/0.17.0
